build-logic/jvm/src/main/kotlin/testng.kotlin-library.gradle.kts

```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 dependencies {
-    testImplementation(platform("org.jetbrains.kotlin:kotlin-bom:1.6.10"))
+    testImplementation(platform("org.jetbrains.kotlin:kotlin-bom:1.6.20"))
```

Comment:
Is it possible to manage this version in the versions.properties file?

Reply:
it doesn't work for me